PyDSS/ResultData.py

```diff
@@ -80,16 +80,17 @@ def updateSubscriptions(self):
 
     def _create_element_list(self, objs, properties):
         elements = []
-        element_names = set()
+        #element_names = set()
         for property_name in properties:
             assert isinstance(property_name, str)
             for name, obj in objs.items():
                 if not obj.IsValidAttribute(property_name):
                     raise InvalidParameter(f"{element_class} / {property_name} {name} cannot be exported")
-                if name not in element_names:
-                    elements.append((name, obj))
-                    element_names.add(name)
-        return elements, element_names
+                #TODO: Will commenting this be an issue ever Dan?
```

Comment:
I was wrong before.  We are using it to control what gets appended to `elements`.  `element_names` doesn't need to be returned, though.